src/devtools/client/debugger/src/components/PrimaryPanes/SourcesTreeItem.js

```diff
@@ -101,32 +101,7 @@ class SourceTreeItem extends Component {
       }
     }
 
-    if (isDirectory(item)) {
-      this.addCollapseExpandAllOptions(menuOptions, item);
```

Comment:
Don't we still need the collapse all/expand all context menu item?

If not we can take the method out as well:

https://github.com/RecordReplay/devtools/blob/74d3a8d8a2f0fab17ed37d36720c7efbcee3cd50/src/devtools/client/debugger/src/components/PrimaryPanes/SourcesTreeItem.js#L148-L164

Reply:
oops. good catch